python/lsst/ts/xml/data/sal_interfaces/Electrometer/Electrometer_Commands.xml

```diff
@@ -67,10 +67,10 @@
   <SALCommand>
     <Subsystem>Electrometer</Subsystem>
     <EFDB_Topic>Electrometer_command_setRange</EFDB_Topic>
-    <Description>Set measurement range, it will use the current unit selected</Description>
+    <Description>Set measurement range, which effects the accuracy of measurements and the max signal that can be measured. The maximum input signal is 105% of the measurement range. It will set the value for the current mode selected. Auto-range will automatically go to the most sensitive (optimized) range to make measurements. This is recommended.</Description>
```

Comment:
Is Auto ranging recommended? 
Above there is text that says one should run the zero point correction, does that get done automagically when using auto-range?

Reply:
I think auto-range is recommended. the zero point correction does not get run automatically.

Reply:
Should a ticket be filed to make this happen? Clearly it's not part of this ticket...